compiler/lsp-tests/src/Main.hs

```diff
@@ -6,10 +6,11 @@
 module Main (main) where
 
 import Control.Lens hiding (List)
+import Control.Monad (forM, forM_, void)
 import Control.Monad.IO.Class
 import DA.Bazel.Runfiles
 import Data.Aeson (toJSON)
-import qualified Data.Text as T
+import qualified Data.Text.Extended as T
```

Comment:
I’d prefer to reduce our dependency on `da-hs-base rather than extend it. Do you actually need this here? I couldn’t spot anything that isn’t already in `Data.Text`.

Reply:
It's for `T.show`. Sure, it's easy to replace.

Reply:
Ah missed that one, yes I would just go for `T.pack . show` here.